scripts/build_utils.sh

```diff
@@ -1081,3 +1081,30 @@ maybe_reset_ci_container() {
         CI_CONTAINER=false
     fi
 }
+
+setup_build_area() {
```

Comment:
Can you rename this to `setup_rpm_build_area()` ?

Reply:
renamed.